packages/@okta/vuepress-site/.vuepress/public/docs/api/postman/event-hooks.json

```diff
@@ -0,0 +1,235 @@
+{
+  "info": {
+    "_postman_id": "bc903859-b682-459f-bfc5-94525fde4543",
+    "name": "Event Hooks",
+    "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+  },
+  "item": [
+    {
+      "name": "List all event hooks",
+      "request": {
+        "method": "GET",
+        "header": [
+          {
+            "key": "Authorization",
+            "value": "SSWS {{apikey}}"
+          }
+        ],
+        "body": {
+          "mode": "raw",
+          "raw": ""
+        },
+        "url": {
+          "raw": "{{url}}/api/v1/eventHooks",
+          "host": [
+            "{{url}}"
+          ],
+          "path": [
+            "api",
+            "v1",
+            "eventHooks"
+          ]
+        }
+      },
+      "response": []
+    },
+    {
+      "name": "Get event hook by id",
+      "request": {
+        "method": "GET",
+        "header": [
+          {
+            "key": "Authorization",
+            "value": "SSWS {{apikey}}"
+          }
+        ],
+        "body": {
+          "mode": "raw",
+          "raw": ""
+        },
+        "url": {
+          "raw": "{{url}}/api/v1/eventHooks/{{eventHookId}}",
+          "host": [
+            "{{url}}"
+          ],
+          "path": [
+            "api",
+            "v1",
+            "eventHooks",
+            "{{eventHookId}}"
+          ]
+        }
+      },
+      "response": []
+    },
+    {
+      "name": "Create new event hook",
+      "request": {
+        "method": "POST",
+        "header": [
+          {
+            "key": "Authorization",
+            "value": "SSWS {{apikey}}"
+          },
+          {
+            "key": "Content-Type",
+            "value": "application/json"
+          }
+        ],
+        "body": {
+          "mode": "raw",
+          "raw": "{\n    \"name\": \"Login And Create User event hook\",\n    \"events\": {\n        \"type\": \"EVENT_TYPE\",\n        \"items\": [\n            \"user.session.start\",\n            \"user.lifecycle.create\"\n        ]\n    },\n    \"channel\": {\n        \"type\": \"HTTP\",\n        \"version\": \"1.0.0\",\n        \"config\": {\n            \"uri\": \"https://example.com\",\n            \"headers\": [\n                {\n                    \"key\": \"Authentication\",\n                    \"value\": \"some-api-key\"\n                },\n                {\n                    \"key\": \"x-any-key\",\n                    \"value\": \"my-header-value\"\n                }\n            ],\n            \"authScheme\": {\n                \"type\": \"HEADER\",\n                \"key\": \"Authorization\",\n                \"value\": \"my-shared-secret\"\n            },\n            \"method\": \"POST\"\n        }\n    }\n}"
```

Comment:
nit : I'd suggest removing or renaming the header key `Authentication`, since it may seem confusing. Ideally these keys will be defined in the `AuthScheme`.

Also, may be better to use url as `https://example.com/eventHooks` to be consistent with other doc examples

Reply:
Ack. Changed it

Reply:
tried to use the payload run the create webhooks request, got 400 error massage as ""The request body was not well-formed.". The format seemed to be valid, possible wrong field is "events". Do we have "items" field?

Reply:
@alexandrazhang-okta Are you using the lastest code? I tested this on the latest master that has Amogh's changes.